src/dashboards/components/dashboard_index/DashboardCard.tsx

```diff
@@ -232,16 +244,16 @@ class DashboardCard extends PureComponent<Props> {
       },
     } = this.props
 
-    let dest = `/${PROJECT_NAME.toLowerCase()}/from/dashboard/${id}`
+    let dashboardUrl = `/orgs/${orgID}/dashboards/${id}`
 
-    if (!isFlagEnabled('boardWithFlows')) {
-      dest = `/orgs/${orgID}/dashboards/${id}`
+    if (isFlagEnabled('boardWithFlows')) {
```

Comment:
changing the order of this `if` statement so that the change that's been flagged is the exceptional codepath for the `if` statement, not the default codepath.